exercises/practice/square-root/.meta/example.jl

```diff
@@ -0,0 +1,18 @@
+function square_root(n::Int64; tol=1e-6)
+    digs = round(Int, -log10(tol))
+    if n ≤ 0
+        throw(DomainError(n, "Can only calculate roots of natural numbers."))
+    end
+    x = initial_guess(n)
+    while abs(x^2 - n) > tol
+        x = 0.5 * (x + n / x)
+    end
+    round(x; digits=digs)
+end
+
+
+function initial_guess(n::Int64)
+    m = Int(floor(log10(n) / 2))
```

Comment:
This isn't important, but this pattern is a bit more idiomatic, I think, and better matches your line 2.

```suggestion
    m = floor(Int, log10(n) / 2)
```